weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooks.ts

```diff
@@ -239,13 +246,14 @@ const useCalls = (
       setCallRes({calls: []});
     };
     getTsClient().callsSteamQuery(req).then(onSuccess).catch(onError);
-  }, [entity, project, deepFilter, limit, opts?.skip, getTsClient]);
+  }, [entity, project, deepFilter, limit, opts?.skip, doRefetch, getTsClient]);
```

Comment:
nit: awkward to have a symbol not used in the effect itself be in the deps list. This is because many linters check for this and it can be removed without much caution. Consider using the variable to ensure it is always included